final/ticket-service-acceptance-tests/src/test/resources/features/bookings.feature

```diff
@@ -0,0 +1,51 @@
+Feature: allows users to book tickets for screenings, and show booking history
+
+  Background:
+    Given the application is started
+    And the prompt containing "Ticket service>" is printed
+    And the user types the "sign up sanyi asdQWE123" command
+    And the user types the "sign up laci asdQWE123" command
+    And the user types the "sign in privileged admin admin" command
+    And the user types the "create room Pedersoli 20 10" command
+    And the user types the "create room Girotti 10 10" command
+    And the user types the "create movie Sátántangó drama 450" command
+    And the user types the "create movie \"Spirited Away\" animation 125" command
+    And the user types the "create screening \"Spirited Away\" Girotti \"2021-03-15 10:00\"" command
+    And the user types the "create screening Sátántangó Pedersoli \"2021-03-15 10:45\"" command
+    And the user types the "sign out" command
+
+  @grade4-requirement
+  Scenario: an authenticated user can book a ticket for a screening
+    Given the user types the "sign in sanyi asdQWE123" command
+    When the user types the "book Sátántangó Pedersoli \"2021-03-15 10:45\" (5,5) (5,6)" command
```

Comment:
Having an unlimited number of parameters looks quite problematic; we should wrap (5,5) (5,6) in \" \".